src/view/prsTreeDataProvider.ts

```diff
@@ -63,6 +70,18 @@ export class PullRequestsTreeDataProvider implements vscode.TreeDataProvider<Tre
 	}
 
 	async getChildren(element?: TreeNode): Promise<TreeNode[]> {
+		if (!this._prManager) {
```

Comment:
Probably not related to this change but when I set the `github.remotes` to `nonsense` and then revert it back, the prManager is still there. But it's a very low pri issue.